curriculum/challenges/spanish/03-front-end-libraries/react/access-props-using-this.props.spanish.md

```diff
@@ -48,7 +48,7 @@ class ReturnTempPassword extends React.Component {
     return (
         <div>
             { /* change code below this line */ }
-            <p>Your temporary password is: <strong></strong></p>
+            <p>Tu contraseña temporal es: <strong></strong></p>
```

Comment:
You have put the code in the wrong section.  Please review the English version.  Also, you should not be translating the code, because a completely new set of tests would have to be written.  For now, just revert this code

Reply:
Sorry about that, I resolve some conflicts and make the changes, is this enough or I need something else?? It's my first PR and I'm still learning how GitHub flow works, Thanks for your patience